src/Bookings.js

```diff
@@ -1,22 +1,26 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Search from "./Search.js";
 import SearchResults from "./SearchResults.js";
 // import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import fakeBookings from "./data/fakeBookings.json";
```

Comment:
Is this fake data being used somewhere? If not I'd recommend removing it so it doesn't confuse you.